src/ImageSharp/Formats/Webp/Lossy/Vp8Encoding.cs

```diff
@@ -668,62 +654,62 @@ public static void FTransformWht(Span<short> input, Span<short> output, Span<int
         // luma 16x16 prediction (paragraph 12.3).
         public static void EncPredLuma16(Span<byte> dst, Span<byte> left, Span<byte> top)
         {
-            DcMode(dst.Slice(I16DC16), left, top, 16, 16, 5);
-            VerticalPred(dst.Slice(I16VE16), top, 16);
-            HorizontalPred(dst.Slice(I16HE16), left, 16);
-            TrueMotion(dst.Slice(I16TM16), left, top, 16);
+            DcMode(dst[..], left, top, 16, 16, 5);
```

Comment:
`I16DC16` seems to be missing here?

Reply:
I'll re read the change.

Reply:
No need for a slice at all so just pass `dst`